src/Microsoft.AspNetCore.SignalR/GetHttpContextExtensions.cs

```diff
@@ -1,16 +1,21 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Http.Connections.Features;
 
 namespace Microsoft.AspNetCore.SignalR
 {
-    public static class HubCallerContextExtensions
+    public static class GetHttpContextExtensions
```

Comment:
It felt unnecessary to create a whole new class just to duplicate this extension method. I'm OK splitting this out into two again if people really don't like this.